app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/principal/injected/Covid19ClaimCheck.scala.html

```diff
@@ -62,23 +68,37 @@
     </ul>
 }
 
-@principal_main_template(title = messages("covid_19_claim_check.title"), form = Some(covid19ClaimCheckForm)) {
+@ariaDescribedBy = {
+@if(covid19ClaimCheckForm("yes-no").error.isDefined) {
+    aria-describedby="yes-no-error"
+}
+}
+
+@mainTemplate(title = messages("covid_19_claim_check.title"), optForm = Some(covid19ClaimCheckForm)) {
+
+    <div class="govuk-form-group @{if (covid19ClaimCheckForm("yes-no").error.isDefined) "govuk-form-group--error"}" @ariaDescribedBy>
+
+        <legend>
+            <h1 class="govuk-heading-l">
+                @messages("covid_19_claim_check.title")
+            </h1>
+        </legend>
 
-    @summaryErrorHelper(covid19ClaimCheckForm)
+        <div class="govuk-form-group govuk-summary-list">@inputContent</div>
 
-    @form(action = postAction) {
-        @yesNoRadioHelper(
+        @form(action = postAction) {
+
+        @radioHelper(
             field = covid19ClaimCheckForm(fieldName),
-            legend = messages("covid_19_claim_check.title"),
-            content = Some(inputContent),
-            options = Seq(RadioOption(optionName = YesNoMapping.option_yes, message = messages("base.yes"), classes = None, conditionalContent = None),
-            RadioOption(optionName = YesNoMapping.option_no, message = messages("base.no"), classes = None, conditionalContent = None)
-            ),
-            parentForm = covid19ClaimCheckForm,
-            inline = true
-        )
-
-        <button id="continue-button" class="button" type="submit">@messages("base.continue")</button>
+            isPageHeading = false,
+            inline = true,
+            radioItems = Seq(
+                RadioItem(value = Some(option_yes), content = HtmlContent(messages("base.yes"))),
+                RadioItem(value = Some(option_no), content = HtmlContent(messages("base.no")))),
```

Comment:
```suggestion
                RadioItem(value = Some(option_no), content = Text(messages("base.no")))),
```
HtmlContent not necessary for the purposes of this radio button

Reply:
Okay